tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -484,3 +484,8 @@ def get_aws_username_from_event(logger, event):
     else:
         user_id = identity['principalId']
     return user_id
+
+
+def unique(seq):
+    return list({k: None for k in seq})
```

Comment:
nit: could this be `list(set(seq))` or `sorted(set(seq))` instead?

Reply:
using a set looses original order